config.yaml

```
@@ -8,32 +8,24 @@
 ---
 Standards:
   -
-    Title: Binary Numbers
-    Description: Binary Numbers
-    UID: 7985c55db967192267ccd8c6db86d270
+    Title: About PSEs
+    Description: About Weekly Problem-Solving Exercises 
+    UID: 1e030b14-342a-49b5-89b5-4472f165cbfb
     SuccessCriteria:
       - success criteria
     ContentFiles:
       -
         Type: Lesson
         UID: a6af8fa6bab71c8b803ecaf34e0984c1
-        Path: /01-binary-numbers/01-binary-numbers.md
-      -
-        Type: Checkpoint
-        UID: 45f708d7a65ca33c0e8e298ae804f9bd
-        Path: /01-binary-numbers/02-binary-numbers-checkpoint.md
+        Path: 00-about-pses/01-about-pses.md
   -
-    Title: Arrays
-    Description: Arrays
-    UID: ff43b8de4f41d5103405ddb62eb8d34e
+    Title: PSE 1
+    Description: PSE 01
+    UID: ef4fcb90-29f4-4f87-b2d7-6c61f61a3d50
     SuccessCriteria:
       - success criteria
     ContentFiles:
-      -
-        Type: Lesson
-        UID: ece6a3da4a290866251b1bd78d1dbb37
-        Path: /02-arrays/01-arrays.md
       -
         Type: Checkpoint
-        UID: b3e5030f89373dae804212918d12cf94
-        Path: /02-arrays/02-arrays-checkpoint.md
+        UID: 1caa645b-5306-4ba3-b37f-c02d11bb680e
+        Path: /01-pse-01/01-wk-02-pse-checkpoint.md
```

Nitpick: I think the naming convention is `01-wk02-pse-checkpoint.md` (no dash between `wk` and `02`)

---

Nitpick: use a suggestion for your nitpicks.  :-P

```suggestion
        Path: /01-pse-01/01-wk02-pse-checkpoint.md
```

---

It'd probably be a more cohesive commit to change the config path and the file name in the same commit